runtime/near-vm/lib/api/Cargo.toml

```diff
@@ -31,14 +31,14 @@ wat = { version = "1.0", optional = true }
 # Dependencies and Development Dependencies for `sys`.
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 # - Mandatory dependencies for `sys`.
-wasmer-vm = { path = "../vm", version = "=2.4.0", package = "near-vm-vm" }
-wasmer-compiler = { path = "../compiler", version = "=2.4.0", package = "near-vm-compiler" }
-wasmer-engine = { path = "../engine", version = "=2.4.0", package = "near-vm-engine" }
-wasmer-types = { path = "../types", version = "=2.4.0", package = "near-vm-types" }
+wasmer-vm = { path = "../vm", package = "near-vm-vm" }
```

Comment:
Hmm… right now I don’t think it would, given that we’re not depending on near-vm yet. But you’re right we’ll have to re-introduce versioning before we can land introducing the actual dependency, in order to keep near-vm-runner publish-able… and this might be quite a mess, but we can cross that bridge when we come to it I guess :)